README.md

```diff
@@ -277,6 +277,10 @@ A comma-separated list of the packages to be installed on the system specified v
 
 #### Users
 
+##### ENABLE_SUDO=true
+
+Makes Pieman install `sudo`. If `ENABLE_USER` is set to `true`, Pieman adds the user `USER_NAME` to `/etc/sudoers`.
```

Comment:
Install `sudo`. If `ENABLE_USER` is set to `true`, Pieman adds the user `USER_NAME` to `/etc/sudoers`.